lms/djangoapps/discussion_api/api.py

```diff
@@ -0,0 +1,104 @@
+"""
+Discussion API internal interface
+"""
+from collections import defaultdict
+from urllib import urlencode
+from urlparse import urlunsplit
+
+from django.core.urlresolvers import reverse
+
+from django_comment_client.utils import get_accessible_discussion_modules
+
+
+def get_course_topics(course, user, build_absolute_uri):
+    """
+    Return the course topic listing for the given course and user.
+
+    Parameters:
+
+    course: The course to get topics for
+    user: The requesting user, for access control
+    build_absolute_uri: a function that takes a relative URI and returns an
+        absolute URI
+
+    Returns:
+
+    A course topic listing dictionary; see discussion_api.views.CourseTopicViews
+    for more detail.
+    """
+    # TODO: This is temporary until the discussion_api thread list view is implemented
```

Comment:
Or to put it another way, are you Ok with releasing this code to production and having developers be exposed to and use the old thread_list url?
Or if not a feature flag, may be prefix the thread url with `WIP` or `Deprecated` for now?